### PR TITLE
Ntr Loadout

### PR DESCRIPTION
**Pull request by Teapug**
i swear if github deletes this im going to jump out a window

### CODE REVIEW

**Comment by Teapug** [comment]
alsol added a jumpskirt for NTR

**Review by Aidenkrz — APPROVED** [review]
This all looks good, now you just have to do the same for blueshield :trollface:

File: Resources/Prototypes/Goobstation/Entities/Clothing/Uniforms/jumpskirts.yml [hunk 1]
```diff
@@ -8,3 +8,14 @@
     sprite: Goobstation/Clothing/Uniforms/Jumpskirt/YTS.rsi
   - type: Clothing
     sprite: Goobstation/Clothing/Uniforms/Jumpskirt/YTS.rsi
+
+- type: entity
+  parent: ClothingUniformSkirtBase
+  id: Centcom_agentSkirt
```
Comment (Piras314) [inline Resources/Prototypes/Goobstation/Entities/Clothing/Uniforms/jumpskirts.yml:14]: You make this mistake in a lot of places, but this needs to be CentcomAgentSkirt, not Centcom_agentSkirt. Look through the rest of your code and make it all PascalCase.

File: Resources/Prototypes/Goobstation/Loadouts/Jobs/Command/nanotrasen_representative.yml
Comment (Piras314) [inline Resources/Prototypes/Goobstation/Loadouts/Jobs/Command/nanotrasen_representative.yml:1]: Same thing all through this.

File: Resources/Prototypes/Goobstation/Loadouts/loadout_groups.yml
Comment (Piras314) [inline Resources/Prototypes/Goobstation/Loadouts/loadout_groups.yml:1]: Here too

File: Resources/Prototypes/Goobstation/Loadouts/role_loadouts.yml
Comment (Piras314) [inline Resources/Prototypes/Goobstation/Loadouts/role_loadouts.yml:1]: And here

File: Resources/Textures/Goobstation/Clothing/Uniforms/Jumpskirt/centcom_agent.rsi/equipped-INNERCLOTHING.png
Comment (Piras314) [inline Resources/Textures/Goobstation/Clothing/Uniforms/Jumpskirt/centcom_agent.rsi/equipped-INNERCLOTHING.png:1]: sprite naming format is good

**Review by Piras314 — CHANGES_REQUESTED** [review]
ID naming format changes required to fit PascalCase. @Teapug

**Comment by Teapug** [comment]
> ID naming format changes required to fit PascalCase. @Teapug

Made requested changes

**Review by Piras314 — APPROVED** [review]
If you send a screenshot of this working in-game it's good to merge

**Comment by Teapug** [comment]
> If you send a screenshot of this working in-game it's good to merge

Small problem, i dont have enough ram to host a server... Could someone else test it?

**Comment by Aidenkrz** [comment]
All is good here now, in the future we also need to move the startinggear out of the pockets and into the backpack, but this is fine for now.

**Pull request merged**